data/mupen64plus.ini

```diff
@@ -10078,6 +10267,10 @@ GoodName=Mortal Kombat Trilogy (U) (V1.2) [t6] (P1 Invincible)
 CRC=83F33AA9 A901D40D
 RefMD5=3DCB15043063BD656A0223C519218CFB
 
+[B6A7B85C123DF4E0B9CC118578DBC9A2]
+GoodName=Mortal Kombat Trilogy (U) (Beta) (1996-05-13) [!]
```

Comment:
Not CRC?

Reply:
The emulator didn't return anything for this file, so no CRC.

Reply:
It's also the same for almost all iQue games and manuals.

Reply:
Even unsupported homebrew generate CRC's... like [Flappy Bird 64](https://github.com/meeq/FlappyBird-N64)